docs/05-technical-reference/ac-01-application-gateway-details.md

```diff
@@ -44,3 +44,9 @@ Application Gateway proxies the following headers while making calls to the regi
 - `X-Forwarded-Client-Cert`
 
 In addition, the `User-Agent` header is set to an empty value not specified in the call, which prevents setting the default value.
+
+## Response rewriting
+
+If during a call to the external system, the target responds with a redirect (`3xx` status code) that points to the URL with the same host and a different path, the `Location` header is modified so that the original target path is replaced with the Application Gateway URL and port, with the sub-path pointing to the called service attached at the end, in this format: `{APP_GATEWAY_URL}:{APP_GATEWAY_PORT}/{APP_NAME}/{SERVICE_NAME}/{SUB-PATH}`.
```

Comment:
```suggestion
If during a call to the external system, the target responds with a redirect (`3xx` status code) that points to the URL with the same host and a different path, the `Location` header is modified so that the original target path is replaced with the Application Gateway URL and port. The sub-path points to the called service attached at the end, in this format: `{APP_GATEWAY_URL}:{APP_GATEWAY_PORT}/{APP_NAME}/{SERVICE_NAME}/{SUB-PATH}`.
```
I'd split this long sentence :D

Reply:
It's the sub-path that's attached and it's important to have it all in once sentence, even though I also hate that it's long :(